src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java

```diff
@@ -165,6 +258,13 @@ public String getDisplayString() {
     return res.toString();
   }
 
+  /**
+   * Handles the preference change events.
+   * Does nothing if newValue is the same as the current value.
+   *
```

Comment:
One empty line as separator should be enough.

Reply:
Fixed